common/metrics/defs.go

```diff
@@ -2070,10 +2070,10 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		PersistenceErrShardOwnershipLostCounter:             {metricName: "persistence_errors_shard_ownership_lost", metricType: Counter},
 		PersistenceErrConditionFailedCounter:                {metricName: "persistence_errors_condition_failed", metricType: Counter},
 		PersistenceErrCurrentWorkflowConditionFailedCounter: {metricName: "persistence_errors_current_workflow_condition_failed", metricType: Counter},
+		PersistenceErrWorkflowConditionFailedCounter:        {metricName: "persistence_errors_workflow_condition_failed", metricType: Counter},
```

Comment:
Do we consider changing metric names to be breaking? This could potentially break some dashboards, no?

Reply:
in terms of functionality, nothing changed.

so i do not consider this metrics change a breaking change